SharedPackages/DataBrokerProtectionCore/Sources/DataBrokerProtectionCore/Pixels/DataBrokerProtectionEventPixels.swift

```diff
@@ -146,9 +146,12 @@ public final class DataBrokerProtectionEventPixels {
         #if os(iOS)
         fireBackgroundTaskSessionMetrics()
         #endif
+
+        #if os(iOS) || DEBUG
+        fireStalledOperationMetrics(for: data)
+        #endif
```

Comment:
Make this fireable in macOS DEBUG mode so we have an easier time testing it & check in the db.